app/src/lib/parser/function_words.js

```diff
@@ -1,8 +1,10 @@
 // prettier-ignore
-export const function_words = [
+export const FUNCTION_WORDS = new Set([
 	'a',
 	'an',
 	'certainly',
+	'continue',
```

Comment:
While 'continue' is in the ontology, it is only used to determine the verb aspect within the analyzer, never as a source concept.